portal-web/test/functional/com/liferay/portalweb/tests/enduser/staging/usecase/RemoteStaging.testcase

```diff
@@ -36,6 +36,53 @@ definition {
 		}
 	}
 
+	@description = "This is a use case for LPP-8122. Verify that when remote publishing, the child pages are not published when they are unselected."
+	@priority = "3"
+	test AssertCanotPublishChildPageWhenUnselected {
+		property test.name.skip.portal.instance = "RemoteStaging#AssertCanotPublishChildPageWhenUnselected";
+
+		task ("Given: two children pages for 'Staging Test Page'") {
+			for (var i : list "1,2") {
+				JSONLayout.addPublicLayout(
+					groupName = "Site Name",
+					layoutName = "Test Child Page ${i}",
+					parentLayoutName = "Staging Test Page");
+			}
+		}
+
+		task ("When: In Remote Publish settings, select only the parent Page and not the child pages and publish to remote live.") {
+			Navigator.gotoSitePage(
+				pageName = "Staging Test Page",
+				siteName = "Site Name");
+
+			Staging.gotoPublishToLive();
+
+			Staging.configurePublishToLive(
+				uncheckChildPages = "true",
+				uncheckChildPagesNameList = "Test Child Page 1,Test Child Page 2");
+
+			Staging.publishToLive(remoteStaging = "true");
+		}
+
+		task ("Then: Assert can't access child pages on the Remote Staged Site") {
+			User.logoutPG();
+
+			User.firstLoginUI(
+				password = "test",
+				specificURL = "http://localhost:9080",
+				userEmailAddress = "test@liferay.com");
+
+			Navigator.gotoSitePage(
+				baseURL = "http://localhost:9080",
+				pageName = "Staging Test Page",
+				siteName = "Remote Site");
+
```

Comment:
Please ensure the parent page is published to remote site. Need to add an assertion for the parent page.